{{namespace}}/triorb:
  ros__parameters:
    triorb_port: '/dev/ttyACM0'
    odom_duration: 0.01
    watchdog_timeout: 2.0
    world_frame: 'map'
    robot_frame: 'base_link'
    state_pub_duration: 0.1
    debug: false

